build_tools/python/e2e_test_framework/models/tf_models.py

```diff
@@ -12,32 +12,32 @@
     id=unique_ids.MODEL_MINILM_L12_H384_UNCASED_INT32_SEQLEN128,
     name="MiniLML12H384Uncased",
     tags=["int32", "seqlen128"],
-    source_type=common_definitions.ModelSourceType.EXPORTED_TF_V2,
+    source_type=common_definitions.ModelSourceType.EXPORTED_MHLO_MLIR,
```

Comment:
This will break the longevity of regression benchmarks because we change the imported config id.

We can either do a backfill, or a workaround is to reuse the `unique_ids.IREE_MODEL_IMPORT_TF_V1_DEFAULT` as the new mhlo import config id. And drop the Tensorflow import configs (we can introduce them later if ever needed). And although we have tensorflow v1 and v2 import config, it was a mistake to use `IREE_MODEL_IMPORT_TF_V1_DEFAULT` for the both configs, which is actually an advantage here because all tensorflow benchmarks use the same import config id and can be transferred to the mhlo import config.

Reply:
Thanks for catching this. I updated to use the `unique_ids.IREE_MODEL_IMPORT_TF_V1_DEFAULT` ID and removed the SavedModel import configs.